Makefile

```diff
@@ -39,6 +39,9 @@ run-portainer:
 run:
```

Comment:
do we need to expand the copyright at the top to 2023?

Reply:
The copyright can be adjusted and standardized in a follow-up PR